lib/bindings/HTTPBindings.js

```diff
@@ -159,7 +159,7 @@ function returnCommands(req, res, next) {
         updates = commandList.map(createCommandUpdate);
         cleanCommands = commandList.map(cleanCommand);
 
-        async.parallel(updates.concat(cleanCommands), function(error, results) {
```

Comment:
`results` seems a reminder of a log that was necessary.

Reply:
In that case the fix should be to use `results` to provide a proper log messages based on it.

Reply:
In this case we continue to rely on ISO 25010 to improve software quality. Mainly:

Rule 1: Unused function parameters should be removed, based on:

    MISRA C++:2008, 0-1-11 - There shall be no unused parameters (named or unnamed) in nonvirtual functions.
    MISRA C:2012, 2.7 - There should be no unused parameters in functions
    CERT, MSC12-C. - Detect and remove code that has no effect or is never executed

So, this is fixed in a2e11f62c9516932f9f4e42afc7a52b915322831

Reply:
We agree that unused parameters should be avoided. However, we are proposing a different solution for this case: instead of removing the unused `results` parameter, let's add a log trace that uses `results` to print some useful information in the logs for the user.

What do you think?

Reply:
Based on the maintainability axis of ISO 25010, I think it is counterproductive to add code to the function to use a variable that is currently deprecated, which makes it less readable. Therefore, before adding code that will not be functional, I prefer to leave the function as it was, with the variable "results".

Reply:
fixed in 9dfbdcb16e08ba11ae9b11dac785a6be24469bf6